docs/fhir-guide/FHIR 101 - Practical Guide.ipynb

```diff
@@ -0,0 +1,2966 @@
+    "    \"title\": \"Kids First Research Participant\",\n",
+    "    \"status\": \"draft\",\n",
+    "    \"publisher\": \"Kids First DRC\",\n",
+    "    \"description\": \"The individual human or other organism.\",\n",
+    "    \"fhirVersion\": \"4.0.0\",\n",
+    "    \"kind\": \"resource\",\n",
+    "    \"abstract\": False,\n",
+    "    \"type\": \"Patient\",\n",
+    "    \"baseDefinition\": \"http://hl7.org/fhir/StructureDefinition/Patient\",\n",
+    "    \"derivation\": \"constraint\",\n",
+    "    \"differential\": {\n",
+    "        \"element\": [\n",
+    "            {\n",
+    "                \"id\": \"Patient\",\n",
+    "                \"path\": \"Patient\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.identifier\",\n",
+    "                \"path\": \"Patient.identifier\",\n",
+    "                \"mustSupport\": True\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.name\",\n",
+    "                \"path\": \"Patient.name\",\n",
+    "                \"max\": \"0\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.telecom\",\n",
+    "                \"path\": \"Patient.telecom\",\n",
+    "                \"max\": \"0\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.address\",\n",
+    "                \"path\": \"Patient.address\",\n",
+    "                \"max\": \"0\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.photo\",\n",
+    "                \"path\": \"Patient.photo\",\n",
+    "                \"max\": \"0\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.contact\",\n",
+    "                \"path\": \"Patient.contact\",\n",
+    "                \"max\": \"0\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.gender\",\n",
+    "                \"path\": \"Patient.gender\",\n",
+    "                \"min\": 1\n",
+    "            }\n",
+    "        ]\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true,
+    "hidden": true
+   },
+   "source": [
+    "#### Anatomy of StructureDefinition"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "We won't cover this in great detail but here are the important parts of the `StructureDefinition`:\n",
+    "\n",
+    "- `resourceType` - every FHIR resource has to declare what type it is\n",
+    "- `baseDefinition` - this points to the `StructureDefinition` that the current one extends. It doesn't have to be one of the base FHIR definitions. It could be a `StructureDefinition`   from any FHIR data model. This is one of the features of FHIR that helps achieve interoperability.\n",
```

Comment:
```suggestion
    "- `baseDefinition` - this points to the `StructureDefinition` that the current one extends. It doesn't have to be one of the base FHIR definitions. It could come from any FHIR data model. This is one of the features of FHIR that helps achieve interoperability.\n",
```